Update CCPP variable pdf files for SCM v4.0 and UFS v1.0 release.

JulieSchramm opened the pull request.

ligiabernardet commented:
Thank you for updating these PDFs. Can you also remove the unnecessary files in these directory and leave just the PDFs?

JulieSchramm commented:
url = https://github.com/NOAA-EMC/fv3atm

branch = release/public-v1

tag = ufs-v1.0.0

repo_url = https://github.com/ufs-community/ufs-weather-model/


On Thu, Mar 26, 2020 at 1:32 PM ligiabernardet <notifications@github.com>
wrote:

> Thank you for updating these PDFs. Can you also remove the unnecessary
> files in these directory and leave just the PDFs?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/ccpp-framework/pull/274#issuecomment-604639869>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA3WNU3VCLMT3QNHDAOFDDTRJOUVNANCNFSM4LUOW65Q>
> .
>

climbfuji commented:
If I was really picky I would say we should move the PDFs one level up to the doc directory, but this requires changing the CCPP prebuild config in fv3atm (not a big deal, though) and various pieces of documentation that refer to those PDFS. Maybe not for the UFS v1.x.y release(s), but when we make the corresponding change for the master branch we should consider this.

Also, can you confirm that without the makefile etc in the DevelopersGuide directory, one can create the PDF by running `pdflatex`? Are the instructions in the CCPP technical documentation consistent with this?

JulieSchramm commented:
Yes, the pdf files can be created from the CCPP_VARIABLE_*.tex files
without the makefile, etc.  There are currently no instructions in the Tech
Doc for making these files.  I added a README.md files with instructions.
I think we can leave the files in this directory, since in the future,
these may be generated by the ccpp_prebuild.py file in some other format
(like rst) and converted to html and pdf in another manner.

On Thu, Mar 26, 2020 at 3:16 PM Dom Heinzeller <notifications@github.com>
wrote:

> If I was really picky I would say we should move the PDFs one level up to
> the doc directory, but this requires changing the CCPP prebuild config in
> fv3atm (not a big deal, though) and various pieces of documentation that
> refer to those PDFS. Maybe not for the UFS v1.x.y release(s), but when we
> make the corresponding change for the master branch we should consider this.
>
> Also, can you confirm that without the makefile etc in the DevelopersGuide
> directory, one can create the PDF by running pdflatex? Are the
> instructions in the CCPP technical documentation consistent with this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/ccpp-framework/pull/274#issuecomment-604690842>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA3WNU4PODYUQ6MBLYINTV3RJPAY7ANCNFSM4LUOW65Q>
> .
>

climbfuji commented on doc/DevelopersGuide/README.md:
```diff
@@ -0,0 +1,12 @@
+
+The PDF files in this directory are generated in the following manner:
+
+When the ``ccpp_prebuild.py`` script is run for the single column model (SCM) or
```
SCM and the ufs-weather-model (not FV3) or any other host model  are all host models. The sentence reads as if SCM wasn't.

climbfuji commented:
Regression tests on hera.intel using NEMSCompsetRun all passed (after removing the redundant "rebuild" flags and changing the regression test baseline back to the official location, i.e. for the code as it is now in the PR). Logs updated in the repository.

[NEMSCompsetRun.log](https://github.com/NCAR/ccpp-framework/files/4405534/NEMSCompsetRun.log)

This PR is ready to merge from my side.
